src/components/InfiniteGrid.js

```diff
@@ -0,0 +1,224 @@
+var React = require('react/addons');
+var PureRenderMixin = React.addons.PureRenderMixin;
+
+/**
+ * Inspired from: https://github.com/ggordan/react-infinite-grid
```

Comment:
Wow. Impressive :smiley:

I think this inspiration link is ok for now, maybe just add `(MIT LICENSE)` to the comment